### PR TITLE
#486- Refactoring integration and processing service

### PR DESCRIPTION
**Pull request by guru-aot**
Refactoring
.integration.service.ts
.processing.service.ts

### CODE REVIEW

**Review by andrewsignori-aot — COMMENTED** [review]
Thanks for the reflector, few questions.
For msfaa-integration
- Should msfaa-integration.service.ts be renamed to msfaa.integration.service.ts (it is not inheriting from the base class because it was done before that, but it should).

For ece-integration
- Should ece-file.service.ts be renamed to processing service?

File: sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/ece-process-integration.scheduler.ts [hunk 1]
```diff
@@ -21,8 +21,6 @@ export class ECEProcessIntegrationScheduler extends BaseScheduler<void> {
   /**
    * Identifies all the applications that have an eligible COE request waiting
    * for a particular institution.
-   * @params job has generatedDate Date in which the assessment for
```
Comment (ann-aot) [inline sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/ece-process-integration.scheduler.ts:24]: job is a parameter right?, maybe you can adjust the comment. instead for removing

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the refactors, looks good.

File: sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts [hunk 2]
```diff
@@ -11,9 +11,9 @@ import {
   ESDCFileResponse,
 } from "./models/e-cert-integration-model";
 import { Injectable } from "@nestjs/common";
-import { ECertIntegrationService } from "./e-cert-integration.service";
-import { ECertFullTimeIntegrationService } from "./e-cert-full-time-integration/e-cert-full-time-integration.service";
-import { ECertPartTimeIntegrationService } from "./e-cert-part-time-integration/e-cert-part-time-integration.service";
+import { ECertIntegrationService } from "./e-cert.integration.service";
+import { ECertFullTimeIntegrationService } from "./e-cert-full-time-integration/e-cert-full-time.integration.service";
+import { ECertPartTimeIntegrationService } from "./e-cert-part-time-integration/e-cert-part-time.integration.service";
 import { ECertFullTimeResponseRecord } from "./e-cert-full-time-integration/e-cert-files/e-cert-response-record";
```
Comment (dheepak-aot) [inline sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts:17]: Same about ECertFileHandler , it must be ECertFileProcessingService right?. Please let me know if I am missing something.

image

File: sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece.processing.service.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import {
 import { getFileNameAsCurrentTimestamp } from "@sims/utilities";
 import { DisbursementScheduleService } from "@sims/integrations/services";
```
Comment (dheepak-aot) [inline sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece.processing.service.ts:9]: This file and class renaming must happen for these Classes and it's corresponding files as well.

```
CRAPersonalVerificationService
ECEFileService
DisbursementReceiptRequestService
MSFAARequestService
MSFAAResponseService
IER12FileService
```

Comment (guru-aot) [reply sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece.processing.service.ts:9]: Yeah they have been renamed, please check

**Review by dheepak-aot — APPROVED** [review]
Thanks for doing the changes @guru-aot . Thanks for the code refactor.

**Pull request merged**